.jenkins/pytorch/test.sh

```diff
@@ -274,6 +274,20 @@ test_inductor_huggingface() {
   python benchmarks/dynamo/check_csv.py -f "$TEST_REPORTS_DIR"/inductor_training_huggingface.csv
 }
 
+test_inductor_huggingface_perf() {
+  # Use test-reports directory under test folder will allow the CI to automatically pick up
+  # the test reports and upload them to S3. Need to use full path here otherwise the script
+  # will bark about file not found later on
+  TEST_REPORTS_DIR=$(pwd)/test/test-reports
+  mkdir -p "$TEST_REPORTS_DIR"
+  # Check inference with --float32
+  python benchmarks/dynamo/huggingface.py --performance \
```

Comment:
We are not monitoring the inference perf on our dashboard. This is nice to have if it doesn't add too much test time.

Reply:
Let me remove the inference perf part, as the total huggingface testing time seems long (1h30m).

Reply:
And I will also remove the inference perf part for timm and torchbench